src/AggregateFunctions/QuantileTDigest.h

```diff
@@ -239,7 +246,8 @@ class QuantileTDigest
         if (summary.empty())
             return std::is_floating_point_v<ResultType> ? NAN : 0;
 
-        compress();
+        if (unmerged > 0)
+            throw Poco::Exception("QuantileTDigest was not finalized");
```

Comment:
Looks like the code should be LOGICAL_ERROR (or assert?). See the one below as well.

Reply:
And the ones in ReservoirSampler.

Reply:
I can make it `LOGICAL_ERROR`.
Just this class didn't use our exceptions, so I used `Poco::Exception` too.